clouddriver-web/config/clouddriver.yml

```diff
@@ -267,6 +267,7 @@ openstack:
       endpoint: ${OS_AUTH_URL:''}
       regions: ${OS_REGIONS:''}
       insecure: ${OS_INSECURE:false}
+      heatTemplatePath: /Users/z001mj7/.spinnaker
```

Comment:
Probably not the best default. In fact, do you want to not have a default at all?